fcrepo-client-impl/src/main/java/org/fcrepo/client/impl/FedoraDatastreamImpl.java

```diff
@@ -93,6 +93,11 @@ public void setGraph( final Graph graph ) {
     }
 
     @Override
+    public String getPropertiesPath() {
+        return path + "/fcr:metadata";
```

Comment:
We should probably use the FedoraJcrTypes.FCR_METADATA constant for this instead of a literal.